flake.nix

```diff
@@ -3,8 +3,11 @@
 
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
   inputs.flake-utils.url = "github:numtide/flake-utils";
+  inputs.nix_2-13.url = "github:NixOS/nix/2.13-maintenance";
+  # FIXME: the tip of the branch fails to build on Hydra, but we need it chached
+  inputs.nix_2-18.url = "github:NixOS/nix/f5f4de6a550327b4b1a06123c2e450f1b92c73b6";
```

Comment:
Since this is hardcoded, it means that it must be manually changed at each release. Who's responsible for that? Is there a guaranty that it'll be done? Or can we automate it like nixos.org do?

Reply:
Regarding the displayed Nix versions, I guess there should be (up to) four rather than three:

- `master`
- The latest release
- The latest release that made it to Nixpkgs unstable (which will often be the latest release, but not always)
- The release in NixOS stable

Reply:
> Since this is hardcoded, it means that it must be manually changed at each release. Who's responsible for that?

For now I'll just do it. 

> Is there a guaranty that it'll be done?


> Or can we automate it like nixos.org do?

We can in principle but I refrained from that for now because rebuilding the Nix manual is quite expensive, and as this very commit shows there is no guarantee we'll get a cache hit on Hydra.

Another alternative would be providing a tarball of the manual as part of the Nix release, then we can pick the latest successful build and extract it here. Then automation  will come for free. But that will take a bit longer, especially if we wanted to backport the tarball builds.